packages/embark/src/lib/core/engine.js

```diff
@@ -49,9 +51,23 @@ class Engine {
       this.ipc.serve();
       return callback();
     }
+
     callback();
   }
 
+  registerCommands() {
+    this.events.setCommandHandler('embark:path', function () {
+      const cb = arguments[arguments.length - 1];
+      const pathParts = Array.from(arguments).splice(0, arguments.length - 1);
+      cb(fs.embarkPath(...pathParts));
+    });
+    this.events.setCommandHandler('dapp:path', function () {
```

Comment:
dappPath is available via the config, we should maybe add embarkPath to the config if not and use that?

And I think the embark object embed the config